src/components/map/RouteMap.svelte

```diff
@@ -46,5 +55,7 @@
 				mapProvider.addStopMarker(stop, stopTime);
 			}
 		}
+
+		currentIntervalId = await fetchAndUpdateVehicles(routeId, mapProvider);
```

Comment:
what happens if `routeId` is null? I see you're grabbing it from `moreTripData?.routeId` on line 37, which suggests it can be null.

Reply:
Since the api requires the routeId, I do not think this can be null, but I did add that it will not draw any buses if there are no trips.


https://developer.onebusaway.org/api/where/elements/trip